frb_rust/src/dart_api_dl/include/internal/dart_api_dl_impl.h

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
This folder (`frb_rust/src/dart_api_dl/include/`) contains a copy of Dart SDK https://github.com/dart-lang/sdk/tree/master/runtime/include folder.

Reply:
only dart_api_dl and its dependencies are used.

Reply:
Get it (as above)

Reply:
So it is quite weird IMHO. Maybe we should at least file an issue at Dart repo, asking why an end user should include "internal" folders. Then after they give positive reply saying it is correct, we can paste the link into this repository's code comments, so future readers are convinced.

Reply:
from dart_api_dl.h
 ```
 * All symbols are postfixed with _DL to indicate that they are dynamically
 * linked and to prevent conflicts with the original symbol.
 *
 * Link `dart_api_dl.c` file into your library and invoke
 * `Dart_InitializeApiDL` with `NativeApi.initializeApiDLData`.
 ```

Reply:
(see comments above)